apps/intellij/src/main/kotlin/dev/nx/console/utils/FindNxConfigurationFiles.kt

```diff
@@ -23,6 +27,7 @@ fun findNxConfigurationFiles(project: Project, includeNxJson: Boolean = true): L
                         ) {
                             paths.add(file)
                         }
+                        ProgressManager.checkCanceled()
```

Comment:
Long running read actions should periodically check for  `ProgressManager.checkCanceled()`